### PR TITLE
Issues/24

### PR DESCRIPTION
**Pull request by ArtemKrayevskiy**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+from library import *
```
Comment (mvchn) [inline main.py:1]: It will be better to import dependencies separately

File: main.py [hunk 2]
```diff
@@ -6,8 +9,20 @@ def print_exception_range(max_value):
     print(f"You should enter a number between 1 and {max_value}")
 
 
-def find_book():
-    pass
+def find_book(library : Library):
+    to_find = str(input("Enter title, author or ISBN "))
+    for i in library.catalog:
```
Comment (mvchn) [inline main.py:14]: some duplicates as line 23

**Review by mvchn — CHANGES_REQUESTED** [review]
Commit https://github.com/trngl8/library-project/pull/58/commits/4213e95cf2ac5d9ab01aada9382e43ac11b8f7c3 should be removed from the branch

File: library.py [hunk 3]
```diff
@@ -54,3 +54,6 @@ def __eq__(self, anohter) -> bool:
         if self.title == anohter.title and self.author == anohter.author and self.year == anohter.year:
             return True
         return False
+    
+    def __repr__(self) -> str:
+        return f"Book {self.title} ritten by {self.author} in {self.year}"
```
Comment (roman-duck) [inline library.py:59]: maybe "written" or other verb

**Review by roman-duck — APPROVED** [review]
function find_book has duplicates and should be refactored

**Review by mvchn — APPROVED** [review]

**Pull request merged**